From e27fb6eeecba03c9816e7f8204d59ed357d365ca Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Thu, 11 Oct 2018 09:13:41 +0300
Subject: [PATCH] uboot [gaudi_pldm]: update clocks

update clocks to latest palladium image
This should be updated for any new image to come (assume got new clock
comapre to its previous).
---

diff --git a/board/habanalabs/gaudi/uboot_pldm.fragment b/board/habanalabs/gaudi/uboot_pldm.fragment
index 5cf64b1..2990a30 100644
--- a/board/habanalabs/gaudi/uboot_pldm.fragment
+++ b/board/habanalabs/gaudi/uboot_pldm.fragment
@@ -1,8 +1,8 @@
-CONFIG_SYS_CLK_FREQ=164000
+CONFIG_SYS_CLK_FREQ=101000
 CONFIG_CONARGS="serial0:1200n8"
 CONFIG_BAUDRATE=1200
 CONFIG_BOOTDELAY=10
-CONFIG_DEBUG_UART_CLOCK=27000
+CONFIG_DEBUG_UART_CLOCK=20200
 CONFIG_BOOTCOMMAND="getfit && bootm $fileaddr#conf@1"
 CONFIG_DEFAULT_DEVICE_TREE="gaudi_pldm"
 CONFIG_SPI_FLASH_MACRONIX=y
